src/interface/gui.py

```diff
@@ -0,0 +1,75 @@
+import streamlit as st
+import os
+import fitz
+
+from streamlit_pdf_viewer import pdf_viewer
+from .initialize import initialize_variables
+
+
+class Gui:

```

Comment:
General note - We have to rewrite this code after projecting session and keep some structure based on it.